### DISCUSSION TITLE
Ultralytics Question

### DISCUSSION THREAD

**Discussion by Jason1-2** [ultralytics/hub#189] [Q&A]
Explanation on my project:
I am working on a project that works with Ultralytics YOLOv8 and I trained my own model to identify "Bottles, Cans, and Cups". I'm using a live camera from a raspberry pi to detect these objects. My lines of code are down below (Note I'm using Thonny for python):

from ultralytics import YOLO
model=YOLO('best.pt')
model.predict(source=0,show=Ture)

These lines of code are able to get a live image from the camera and identify if the object as "Bottle, Can, or Cup". The results of camera are some what printed in red at the bottom of the page in "Shell". 

0: 480x640 1 Bottle, 583ms

This is the type info coming from my camera. The "0" is the name of the camera, the 480x640 is the size, 1 Bottle is what it's identifying, and 583ms is when it happened. Note this gets printed out frame by frame.

My problem is that I can't use IF statements on that info, this is because "0: 480x640 1 Bottle, 583ms" is coming from somewhere else not on my raspberry pi.

My question is, where is that info coming from, and can I have access that data to create statements and eventually create an output signal. I know there is coding outside of my own, is there any chance I can find out where each frame is coming from? 
Thank you,
Jason

**Comment by glenn-jocher** [comment]
@Jason1-2 use `results = model(img)` to return results and then handle them however you want to. See https://docs.ultralytics.com/modes/predict/#working-with-results for details.
